dartbrcode/README.md

```diff
@@ -0,0 +1,276 @@
+# node-brcode
+
+Dart wrapper of `brcode` to parse and emit [PIX BR Code](https://www.bcb.gov.br/content/estabilidadefinanceira/spb_docs/ManualBRCode.pdf).
+
+## Usage
+
+1. Include `dartbrcode` in `pub.yaml`
+```yaml
+dependencies:
+  dartbrcode: ^0.1.0
+```
+2. Copy `libbrcode.*` from [brcode](https://github.com/naomijub/brcode/tree/master/clj-brcode) to your Node project root:
+  - for linux copy `libbrcode.so`.
+  - for macos copy `libbrcode.dylib`.
+  - `cargo build --release` project from [git](https://github.com/naomijub/brcode) and copy the `libbrcode.*` from `target/release/libbrcode.*` to your Dart project root. Mobile in the section `Building for mobile`.
+  - Shellscript to get files from release:
+  **So**
+  ```sh
+  curl -s https://api.github.com/repos/naomijub/brcode/releases/latest \
```

Comment:
I think there's a way to retrieve it using the JSON API.

Reply:
Will look at it later, thanks